tests/unittests/test_ds_identify.py

```diff
@@ -807,6 +830,19 @@ def _print_run_output(rc, out, err, cfg, files):
             'dev/vdb': 'pretend iso content for cidata\n',
         }
     },
+    'NoCloud-fbsd': {
```

Comment:
@smoser i naïvely assumed this would add a test case

Reply:
still no test, right?

Reply:
gimme a day or two, while i work my way thru https://pythontesting.net/start-here/